en/docs/get-started/about-this-release.md

```diff
@@ -12,227 +12,58 @@ For more information on WSO2 API Manager, see the [overview]({{base_path}}/getti
 
 ## New features
 
-#### API Governance
+??? note "JDK 17 support"
+    You can now run our products with JDK 17.
 
-??? note "Management of real-time APIs on Solace"
+??? note "OpenTelemetry Support"
+    OpenTelemetry combines OpenTracing and OpenCencus which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics and logs are collected, transmitted and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends
```

Comment:
```suggestion
    OpenTelemetry combines OpenTracing and OpenCencus, which is a single set of APIs and libraries that standardize how telemetry data such as traces, metrics, and logs are collected, transmitted, and managed. It offers a safe, vendor-neutral specification for instrumentation so that information about the selection can be communicated to various backends.
```